compiler/ilgen/OMRVirtualMachineState.cpp

```diff
@@ -20,15 +20,63 @@
  *******************************************************************************/
 
 #include "ilgen/VirtualMachineState.hpp"
+#include "ilgen/IlBuilder.hpp"
+#include "compile/Compilation.hpp"
 
-namespace TR { class IlBuilder; }
 class TR_Memory;
 
 template <class T> class List;
 template <class T> class ListAppender;
 
+
+void
+OMR::VirtualMachineState::commit(TR::IlBuilder *b)
```

Comment:
That's a good point. My intention was to document that convention in a separate document, instead of the code. The convention, while currently only useful here, is generally applicable to other parts of the API. However, I agree that it really should be in the commit message as well. Will fix.